src/parameterized_tendencies/les_sgs_models/anisotropic_minimum_dissipation.jl

```diff
@@ -0,0 +1,329 @@
+
+    # Scaled Derivatives ∂̂ᵢ = Δ₍ᵢ₎∂ᵢ
+    ᶜ∂̂u_uvw = @.ᶜtemp_UVWxUVW = Δ_h * Geometry.project(axis_uvw, gradₕ(ᶜu_uvw))
+    @. ᶜ∂̂u_uvw += ᶜΔ_z * Geometry.project(axis_uvw, ᶜgradᵥ(ᶠu_uvw))
+
+    ᶠ∂̂u_uvw = @.ᶠtemp_UVWxUVW = Δ_h * Geometry.project(axis_uvw, gradₕ(ᶠu_uvw))
+    @. ᶠ∂̂u_uvw += ᶠΔ_z * Geometry.project(axis_uvw, ᶠgradᵥ_uvw(ᶜu_uvw))
+
+    ᶜ∂ₖuᵢ∂ₖuⱼ = @. lazy(ᶜ∂̂u_uvw * adjoint(ᶜ∂̂u_uvw))
+    ᶠ∂ₖuᵢ∂ₖuⱼ = @. lazy(ᶠ∂̂u_uvw * adjoint(ᶠ∂̂u_uvw))
+    ᶜ∂ₗuₘ∂ₗuₘ = @. lazy(CA.norm_sqr(∇ᶜu_uvw))
+
+    # AMD eddy viscosity
+    # no defined trace method in climacore for axistensors? 
+    ᶜνₜ = @. ᶜtemp_scalar = max(FT(0), - c_amd * 
+                            ((ᶜ∂ₖuᵢ∂ₖuⱼ * ᶜS).components.data.:1 + 
+                             (ᶜ∂ₖuᵢ∂ₖuⱼ * ᶜS).components.data.:5 + 
+                             (ᶜ∂ₖuᵢ∂ₖuⱼ * ᶜS).components.data.:9)/ max.(eps(FT), ᶜ∂ₗuₘ∂ₗuₘ))
+    ᶠνₜ = @. ᶠtemp_scalar = ᶠinterp(ᶜνₜ)
+
+    # Subgrid-scale momentum flux tensor, `τ = -2 νₜ ∘ S`
+    ᶜτ_amd = @. lazy(-2 * ᶜνₜ * ᶜS)
+    ᶠτ_amd = @. lazy(-2 * ᶠνₜ * ᶠS)
+
+    ## Momentum tendencies
+    ᶠρ = @. p.scratch.ᶠtemp_scalar = ᶠinterp(Y.c.ρ)
+    @. Yₜ.c.uₕ -= C12(wdivₕ(Y.c.ρ * ᶜτ_amd) / Y.c.ρ)
+    @. Yₜ.f.u₃ -= C3(wdivₕ(ᶠρ * ᶠτ_amd) / ᶠρ)
+
+    ## Total energy tendency
+    ᶜh_tot = @. lazy(
+        TD.total_specific_enthalpy(
+            thermo_params,
+            ᶜts,
+            specific(Y.c.ρe_tot, Y.c.ρ),
+        ),
+    )
+    ∇h_tot = @. lazy(Geometry.project(axis_uvw, gradₕ(ᶜh_tot)))
+    ∂̂h_tot = @. lazy(Δ_h * Geometry.project(axis_uvw, gradₕ(ᶜh_tot)))
+    ᶜD_amd = @. ᶜtemp_scalar = max(FT(0), -c_amd * 
+                               ((ᶜ∂̂u_uvw * ∂̂h_tot ⊗ ∇h_tot).components.data.:1 +
+                               (ᶜ∂̂u_uvw * ∂̂h_tot ⊗ ∇h_tot).components.data.:5 + 
+                               (ᶜ∂̂u_uvw * ∂̂h_tot ⊗ ∇h_tot).components.data.:9) / 
+                               max(eps(FT), CA.norm_sqr(gradₕ(ᶜh_tot))))
+    @. Yₜ.c.ρe_tot += wdivₕ(Y.c.ρ * ᶜD_amd * gradₕ(ᶜh_tot))
+
+    # Tracer diffusion and associated mass changes
+    foreach_gs_tracer(Yₜ, Y) do ᶜρχₜ, ᶜρχ, ρχ_name
+        ᶜχ = @. lazy(specific(ᶜρχ, Y.c.ρ))
+        ∇ᶜχ = @. lazy(Geometry.project(axis_uvw, gradₕ(ᶜχ)))
+        ∂̂ᶜχ = @. ᶜtemp_UVW = Δ_h * Geometry.project(axis_uvw, gradₕ(ᶜχ)) 
+        @. ∂̂ᶜχ += @. ᶜΔ_z * Geometry.project(axis_uvw, ᶜgradᵥ(ᶠinterp(ᶜχ)))
+        ᶜD_amd = @. ᶜtemp_scalar = max(FT(0), -c_amd * 
+                                   ((ᶜ∂̂u_uvw * ∂̂ᶜχ ⊗ ∇ᶜχ).components.data.:1 +
+                                   (ᶜ∂̂u_uvw * ∂̂ᶜχ ⊗ ∇ᶜχ).components.data.:5 + 
+                                   (ᶜ∂̂u_uvw * ∂̂ᶜχ ⊗ ∇ᶜχ).components.data.:9) / 
+                                   max(eps(FT), CA.norm_sqr(gradₕ(ᶜχ))))
+        ᶜρχₜ_diffusion = @. lazy(wdivₕ(Y.c.ρ * ᶜD_amd * gradₕ(ᶜχ)))
+        @. ᶜρχₜ += ᶜρχₜ_diffusion
+        # Rain and snow does not affect the mass
+        if ρχ_name == @name(ρq_tot)
```

Comment:
Not sure if thats what we want in the end. I'll change it in case the results are better with all contributions in.